app/views/device_stories/show.html.erb

```diff
@@ -47,5 +49,40 @@
   </div>
 </div>
 
+<div>
+  <h2>Comments: <br></h2>
+  <table class="table table-striped">
+    <tbody>
+    <%- if @device_story.device_story_comments.empty? %>
+      No Comments yet.
+    <%- else %>
+      <%- @device_story.device_story_comments.each do |comment| %>
+        <tr>
+          <td>
+            <b><%= link_to comment.user.name , comment.user %></b> <div style="display: inline-block; color: dimgrey; font-size: small">(<%= time_ago_in_words(comment.created_at) %> ago)</div> <br>
+            <%= comment.content %>
+            <%- if user_signed_in? -%>
+              <%- if (comment.user_id == current_user.id) || moderator?(current_user) %>
+                <div style="float:right"> <%= link_to 'Delete', device_story_device_story_comment_path(@device_story, comment), method: :delete %></div>
+              <%- end %>
+            <%- end %>
+          </td>
+        </tr>
+      <%-end %>
+    <%- end %>
+    </tbody>
+  </table>
+</div>
+<div style="margin-left: 5px;min-width: 500px">
+  <%- if user_signed_in? %>
+    <%= form_with model: DeviceStoryComment, url: device_story_device_story_comments_path(@device_story), local: true, method: 'post' do |form| %>
+      <%= form.label :content, current_user.name %>
+      <%= form.text_area :content, :rows => 3, style: 'width:97%;' %>
+      <%= form.hidden_field :user_id, :value => current_user.id %>
```

Comment:
We shouldn't have this here since anyone could override it. Move it to `DeviceStoryCommentsController#create`